h2o-core/src/main/java/hex/Model.java

```diff
@@ -2808,4 +2808,22 @@ public ModelDescriptor modelDescriptor() {
     };
   }
 
+  /**
+   * Convenience method to find out if featureName is used for prediction, i.e., if it has beta == 0 in GLM,
+   * it is not considered to be used.
+   * This is mainly intended for optimizing prediction speed in StackedEnsemble.
+   * @param featureName
+   */
+  public boolean isFeatureUsed(String featureName) {
```

Comment:
Good point! I would probably prefer `isFeatureUsedInPredict`. Also, I added `getUsedFeatures` method that basically lists the features and uses the `isFeatureUsed` as a predicate; just in case it would be useful for someone as @michalkurka suggested. So I will* rename it to something like `getFeaturesUsedInPrediction`. Wdyt?

* Probably after easter; there is still some polishing needed in other parts of the code...